BoxContentSDK/BoxContentSDK/Operations/BOXAPIThumbnailDownloadDataOperation.m

```diff
@@ -0,0 +1,13 @@
+//
+//  BOXAPIThumbnailDownloadOperation.m
+//  BoxContentSDK
+//
+//  Created by Clement Rousselle on 3/16/16.
+//  Copyright © 2016 Box. All rights reserved.
+//
+
+#import "BOXAPIThumbnailDownloadDataOperation.h"
```

Comment:
I would suggest just adding something like a "isThumbnailDownloadOperation" property to BOXAPIDataOperation, rather than creating an empty subclass.